test/Microsoft.IdentityModel.JsonWebTokens.Tests/JsonWebTokenHandler.DecryptTokenTests.cs

```diff
@@ -193,10 +194,42 @@ public static TheoryData<TokenDecryptingTheoryData> JsonWebTokenHandlerDecryptTo
                             ValidationFailureType.TokenDecryptionFailed,
                             typeof(SecurityTokenDecryptionFailedException),
                             null),
+                    },
+                    new TokenDecryptingTheoryData
```

Comment:
This needs another test case where `unwrappedKeys` is 0 similar to AlgorithmMatch in the other test class.